src/quickstarts/openai/README.md

```diff
@@ -0,0 +1,25 @@
+# Quickstarts
+
+These samples demonstrates how to use OpenAI with `gpt-3.5-turbo` and `dall-e-3` models, from a simple .NET 8.0 console application. It consists of a simple console applications, running locally, that will send request to the OpenAI service. 
+
+## Prerequisites
```

Comment:
Are we able to add a document that teaches the user about "You exceeded your current quota" error?
The user might hit this issue after they have account and  ApI key.
Adding this  might help the user learn common OpenAI API related errors 
https://help.openai.com/en/collections/3808446-api-error-codes-explained

Reply:
For now, I'd like to keep these docs simple and focused on the sample, but let's keep a watch for customer feedback. Ideally they find the document you mentioned here, so we don't have to duplicate the info in our docs.